user/yaml/README.md

```diff
@@ -0,0 +1,327 @@
+kubectl exec -n kubestellar $(kubectl get pod -n kubestellar --selector=app=kubestellar-server -o jsonpath={.items[0].metadata.name}) -- tar cf - "/home/kubestellar/kubestellar" | tar xf - --strip-components=2
+```
+
+As an alternative the bootstrap script can be used with the `--deploy false` option as follows:
+
+```shell
+bash <(curl -s https://raw.githubusercontent.com/kubestellar/kubestellar/main/bootstrap/bootstrap-kubestellar.sh) --deploy false
+```
+
+The `admin.kubeconfig` to access **KubeStellar** from outside the cluster can be obtained by:
+
+```shell
+kubectl cp -n kubestellar $(kubectl get pod -n kubestellar --selector=app=kubestellar-server -o jsonpath={.items[0].metadata.name}):/home/kubestellar/.kcp/external.kubeconfig ./admin.kubeconfig
+```
+
+or
+
+```shell
+kubectl get secrets kubestellar -n kubestellar -o 'go-template={{index .data "external.kubeconfig"}}' | base64 --decode > ./admin.kubeconfig
+```
+
+Then, the following environment variables need to be set:
+
+```shell
+export KUBECONFIG=$PWD/admin.kubeconfig
+export PATH=$PATH:$PWD/kcp-plugins/bin:$PWD/kubestellar/bin
+```
+
+If necessary, add the **KubeStellar** ingress, *e.g.* `kubestellar.svc.cluster.local`, to the `/etc/hosts` files:
+
+```text
+127.0.0.1       kubestellar.svc.cluster.local
+```
+
+Now we can use use **KubeStellar** in the usual way:
+
+```shell
+$ kubectl ws tree
+.
+└── root
+    ├── compute
+    └── espw
+```
+
+## Access **KubeStellar** from another pod in the same `kubestellar` namespace
+
+Any pod in the same namespace can access **KubeStellar** by retrieving the `admin.kubeconfig` via one of the following ways:
+
+- the `kubestellar` secret in the `kubestellar` namespace or any other namespace listed in `SECRET_NAMESPACES` environment variable;
+- directly from the `kubestellar` pod in the `kubestellar` namespace at the location `/home/kubestellar/.kcp/external.kubeconfig`.
+
+Obviously `kubectl`, **kcp** plugins, and **KubeStellar** executables are also needed.
+
+In this example, we create a `kubestellar-client` pod based on the same image of `kubestellar-server` since it already contains the required executables listed above:
+
+```shell
+$ kubectl apply -f kubestellar-client.yaml
+deployment.apps/kubestellar-client created
+
+$ kubectl get pods
+NAME                                  READY   STATUS    RESTARTS   AGE
+kubestellar-client-5dcd55b4c7-cvz6j   1/1     Running   0          32s
+kubestellar-server-566f5cb54d-mmp8p   1/1     Running   0          58m
+```
+
+Now, let us log into the pod:
+
+```shell
+kubectl exec -n kubestellar $(kubectl get pod -n kubestellar --selector=app=kubestellar-client -o jsonpath={.items[0].metadata.name}) -it -- bash
+```
+
+From within the pod:
+
+```shell
+[kubestellar@kubestellar-client-7c7d46cf77-sk2d4 /]$ kubectl ws tree
+.
+└── root
+    ├── compute
+    └── espw
+```
+
+## Add a new cluster to **KubeStellar** inventory
+
+A syncer can also be generated using pip commands as in the following example:

```

Comment:
"pip"?

Isn't this specific to the " ## Access **KubeStellar** service from the host OS" usage scenario?

Reply:
pipe